### ISSUE TITLE
PHP: GPBWrapperUtils not found in Google\Protobuf\Internal

### ISSUE THREAD

**Issue by AxiosCros** [protocolbuffers/protobuf#5879]
Version： protoc v3.7.0
Language: PHP

not found "`GPBWrapperUtils`" in [`php/src/Google/Protobuf/Internal`](https://github.com/protocolbuffers/protobuf/tree/master/php/src/Google/Protobuf/Internal), 
when use Google\Protobuf\Internal\GPBWrapperUtils in php code.

here for example : 
[php/src/Google/Protobuf/Internal/DescriptorProto.php](https://github.com/protocolbuffers/protobuf/blob/master/php/src/Google/Protobuf/Internal/DescriptorProto.php) 

``` php
<?php
# Generated by the protocol buffer compiler.  DO NOT EDIT!
# source: google/protobuf/descriptor.proto
namespace Google\Protobuf\Internal;
use Google\Protobuf\Internal\GPBType;
use Google\Protobuf\Internal\GPBWire;
use Google\Protobuf\Internal\RepeatedField;
use Google\Protobuf\Internal\InputStream;
use Google\Protobuf\Internal\GPBUtil;

use Google\Protobuf\Internal\GPBWrapperUtils;  // <====== where is it?
```

**Comment by AxiosCros** [comment]
@TeBoring 

if "never provided that class" , so why use it in [`php/src/Google/Protobuf/Internal/DescriptorProto.php`](https://github.com/protocolbuffers/protobuf/blob/master/php/src/Google/Protobuf/Internal/DescriptorProto.php) . 

I want an answer that can solve the problem.
"I think we never provided that class." is not the answer I wanted.

another issue [#5874](https://github.com/protocolbuffers/protobuf/issues/5874)

**Comment by TeBoring** [comment]
You are using ```GPBWrapperUtils``` by your own? Or just find it when reported by other files which depends on it?

**Comment by AxiosCros** [comment]
I'm using `protoc` to generate php code . 
`use Google\Protobuf\Internal\GPBWrapperUtils;` this code is used in the all of generated code.

If you don't believe me, you can try that for yourself.

``` shell
protoc *.proto --php_out=./php_output
```

**Comment by AxiosCros** [comment]
the content of my proto file is following :
``` 
syntax = "proto3";

option php_namespace = "services\\Aegis";

message AegisCommon {
    string version = 10;
    string code = 20;
    string service_code = 30;
    string global_endpoint = 40;
    enum ApiStyle {
        rpc = 0;
        roa = 1;
    }
    ApiStyle api_style = 50;

    enum Action {
        AutoUpgradeSas = 0;
    }

    Action action_name = 60;
}
```

**Comment by TeBoring** [comment]
I think this is a mistake in our implementation. I will send a PR to remove it.

**Comment by AxiosCros** [comment]
> I think this is a mistake in our implementation. I will send a PR to remove it.

thanks.  please Let me know when it's resolved and which version solves the problem.

**Comment by TeBoring** [comment]
Fixed in #5880. Will release in 3.7.1 release.
